### ISSUE TITLE
Symbol to re-hide filtered post is an eye - it's confusing

### ISSUE THREAD

**Issue by Cassolotl** [mastodon/mastodon#25813]
### Pitch

<img width="622" alt="a2cd1f56eb36144a" src="https://github.com/mastodon/mastodon/assets/6607512/7d33a9b3-982e-48d0-a18d-e9974617d2f8">

This is a post that was hidden because I'm filtering a word in it.

The symbol to re-hide it is an eye, but it should be an eye with a line through it, like this:

<img width="418" alt="Screenshot 2023-07-07 at 22 45 58" src="https://github.com/mastodon/mastodon/assets/6607512/593d29e9-6eab-4c02-9c42-24a2896f4d9d">
<img width="283" alt="Screenshot 2023-07-07 at 22 46 49" src="https://github.com/mastodon/mastodon/assets/6607512/aa87f93a-2f4e-42ea-bcf7-6f81df62cf0a">

### Motivation

When I first saw the eye symbol I was confused. I was already looking at the post, what would the eye do? I'm already looking at the post, would this button make me be able to see it even more somehow?

I think an eye with a line through it would be more intuitive.

I think part of why it's so confusing is because the "show" and "hide" functions are presented very differently. The "show" function is inline linked text, and the "re-hide" button is a symbol beneath the post among other functions (e.g. like, reblog). It throws you a bit.

**Comment by timtfj** [comment]
1. It was only through reading this post that I realised there even _is_ a way to re-hide a filtered post! I've been resorting to re-loading the page. So it's definitely not obvious, at least to me.
2. Having tried using it just now, I agree the crossed-out version would be more explanatory.

It seems to me the eye icon is confusing in every interface that uses it—there's always confusion as to whether it indicates a current state or the state that will result from tapping it, and the meaning varies from app to app.

**Comment by anthonygedeon** [comment]
I agree with @timtfj, if this issue is approved for a pr then I'll gladly handle this issue

**Comment by Cassolotl** [comment]
> It was only through reading this post that I realised there even _is_ a way to re-hide a filtered post! I've been resorting to re-loading the page. So it's definitely not obvious, at least to me.

I think it might be new with the latest update, if that is at all reassuring!

**Comment by Cassolotl** [comment]
Still present in 4.3.
